### DISCUSSION TITLE
[Bug]: Live view stuck as JSMPEG or doesn't load after disabling microphone

### DISCUSSION THREAD

**Discussion by bakedhorse** [blakeblackshear/frigate#11925] [Report a Bug] [answered]
### Describe the problem you are having

Using the microphone on live view for a minute or more, then switching it off results in the Live View using jsmpeg or is stuck with nothing on-screen. Guessing this is a "stall"?

I tried this on both Brave and Firefox, occurred on both. I tried on mobile, this problem does happen in the browser. However this doesn't occur on the Home Assistant App, but does when going to the HA website in the browser.

### Steps to reproduce

1. Setup a camera with two-way audio support
2. Enable microphone on live view for a minute (if even that)
3. Disable the microphone.
4. Now the stream is stuck as jsmpeg or is completely blank.

### Version

0.14.0-2d4d158

### Frigate config file

```yaml
Not relevant.
```

### Relevant log output

```shell
None.
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Mixed

### Camera make and model

Not relevant.

### Any other information that may be helpful

None.

**Comment by NickM-27** [comment, marked as answer]
Thanks for the report, fixed in https://github.com/blakeblackshear/frigate/pull/11931
